src/kernel/rational/givratmisc.C

```diff
@@ -48,20 +48,16 @@ const Integer ceil(const Rational &x)
   return q;
 }
 
-const Integer round(const Rational& x)
+const Integer round(const Rational& x)  // GV Jeu  2 aoû 2018 16:38:58 CEST
 {
   Integer q;
   Integer r;
-  Integer::divmod(q, r, x.num, x.den);
-  r <<= 1;
-  if (absCompare(r,x.den) < 1)
-  {
-    if (sign(x.num) >= 0)
+  Integer::divmod(q, r, abs(x.num), abs(x.den));
+  r <<= 1; 
+  if (absCompare(r,x.den) > 0) // GV was < 0, and changed with abs
       q += 1;
-    else
-      q -= 1;
-  }
-  return q;
+  return sign(x)*q;
+
```

Comment:
This patch fixes `round(Rational(1,3))` (giving 0 as expected), which was wrong beforehand.
However it gives 0 for `round(Rational(1,2))`, which should be 1 (as `std::round(0.5)`).

Basically, I think we should also follow standard C++ behavior: http://www.cplusplus.com/reference/cmath/round/.

I am currently writing tests for Rational, in order to ensure consistency in future works.